packages/runner-ct/cypress/plugins/index.js

```diff
@@ -22,6 +27,41 @@ function injectStylesInlineForPercyInPlace (webpackConfig) {
  */
 module.exports = (on, config) => {
   on('task', percyHealthCheck)
+  on('task', {
+    compareImages (screenshotName) {
+      const readAsync = (p) => {
+        return new Promise((res, rej) => {
```

Comment:
We can use more stable thing to compare images https://github.com/dmtrKovalenko/odiff or at least pixelmatch.

Reply:
Neat, I will update the tests to use this.